icepyx/core/read.py

```diff
@@ -668,7 +680,7 @@ def _build_dataset_template(self, file):
 
     def _read_single_grp(self, file, grp_path):
         """
-        For a given file and variable group path, construct an Intake catalog and use it to read in the data.
+        For a given file and variable group path, construct an an xarray Dataset.
```

Comment:
Typo: remove duplicate 'an'.
```suggestion
        For a given file and variable group path, construct an xarray Dataset.
```